File: Lbs.UI/src/app/interfaces/message.ts
import { Timestamp } from 'rxjs/Rx';

export interface Message {
    id: string;
    imei: string;
    actual_date: Timestamp<string>;
    latitude: number;
    longitude: number;
    direction: number;
    odometer: number;
    speed: number;
    analog: number;
    eventcode: number;
    textm: number;
    fuel: number;
    temp2: number;
    voltage: number;
}

export interface IMessageData {
    date: string
    text: string
}

export interface INotifications {
    actual_date: any
    imei: any
    speed: any
    fuel: any
    message: any
}

File: SignalR-Hub/SignalR-Hub/Message.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace SignalRHub
{
    public class Message
    {
        public string id { get; set; }
        public string imei { get; set; }
        public DateTime actual_date { get; set; }
        public decimal latitude { get; set; }
        public decimal longitude { get; set; }
        public decimal direction { get; set; }
        public decimal odometer { get; set; }
        public decimal speed { get; set; }
        public decimal temperature { get; set; }
        public decimal fuel { get; set; }
        public decimal voltage { get; set; }
    }
    
}
